### PR TITLE
Make /pi command Broadcastable

### PR DESCRIPTION
**Pull request by DeltaCoderr**

### CODE REVIEW

File: server/chat-commands/info.ts [hunk 1]
```diff
@@ -2465,6 +2465,7 @@ export const commands: ChatCommands = {
 
 	'!pi': true,
 	pi(target, room, user) {
+		if (!this.runBroadcast(true)) return false;
```
Comment (xfix) [inline server/chat-commands/info.ts:2468]: Why is it ignoring cooldown?

Comment (DeltaCoderr) [reply server/chat-commands/info.ts:2468]: Oh, sorry forgot about that.

File: server/chat-commands/info.ts [hunk 2]
```diff
@@ -2465,6 +2465,7 @@ export const commands: ChatCommands = {
 
 	'!pi': true,
 	pi(target, room, user) {
+		if (!this.runBroadcast()) return false;
```
Comment (xfix) [inline server/chat-commands/info.ts:2468]: I'm honestly not sure, should commands return `false` or not? This is a question to @Zarel.

Comment (Zarel) [reply server/chat-commands/info.ts:2468]: It's complicated. I don't mind them returning `false`.

**Pull request merged**